crates/utils/re_byte_size/src/std_sizes.rs

```diff
@@ -67,6 +67,19 @@ impl<K: SizeBytes, V: SizeBytes, S> SizeBytes for HashMap<K, V, S> {
     }
 }
 
+impl<K: SizeBytes, S> SizeBytes for HashSet<K, S> {
+    #[inline]
+    fn heap_size_bytes(&self) -> u64 {
+        // NOTE: It's all on the heap at this point.
+
+        if K::is_pod() {
+            (self.len() * std::mem::size_of::<K>()) as _
+        } else {
+            self.iter().map(SizeBytes::total_size_bytes).sum::<u64>()
```

Comment:
that's incorrect: in the non-pod case you want to _add_ this to `(self.capacity() * std::mem::size_of::<K>()) as _`

Reply:
oh. a lot of the implementaitons have this bug. Guess I'll look into cleaning that up